transforms/globals-to-ember-data-imports.js

```diff
@@ -157,6 +157,76 @@ function transform(file, api /*, options*/) {
     return dsUsages.filter(isDSGlobal(globalDS)).paths();
   }
 
+  /*
+   * loops through all modules and replaces literal path if necessary
+   * 'ember-data/model' -> '@ember-data/model'
+   */
+  function updateExistingLiteralPaths(root, module, mappings) {
+    let foundMapping = mappings[module.local];
+
+    if (foundMapping) {
+      let newSource = foundMapping.source;
+      if (module.source !== newSource) {
+        root
+          .find(j.ImportDeclaration, {
+            source: {
+              type: 'Literal',
+              value: module.source
+            }
+          })
+          .find(j.Literal)
+          .forEach(importLiteral => {
+            j(importLiteral).replaceWith(j.literal(newSource));
+          });
+      }
+    }
+  }
+
+  /*
+   * After modifying existing sources to their new paths, we need
+   * to make sure we clean up duplicate imports
+   */
+  function cleanupDuplicateLiteralPaths() {
```

Comment:
I wanted to use the existing `registry.modules` somehow, but at this point in the codemod, they might contain old information.  So in this solution, I went with iterating over the imports (post updating their source) and adding to the existing import found.